packages/analytics-browser/src/plugins/network-connectivity-checker.ts

```diff
@@ -33,6 +33,15 @@ export const networkConnectivityCheckerPlugin = (): BeforePlugin => {
   };
 
   const setup = async (config: BrowserConfig, amplitude: BrowserClient) => {
+    if (typeof navigator === 'undefined') {
+      config.loggerProvider.debug(
+        'Network connectivity checker plugin is disabled because navigator is not available.',
+      );
+      config.offline = true;
```

Comment:
Is there a reason this needs to be defaulted to true? If navigator isn't available, then we want to default to assuming there's internet connectivity? Offline mode is mostly best effort.

Reply:
Good point. FIxing now